### PR TITLE
Installing Q# for other IDEs

### LINKED ISSUE #883: Missing instructions on running Q# from IDEs other than VS/VS Code
There used to be a section with instructions on how to run Q# from pure command line, regardless of the IDE, but it is gone. Based on user feedback, it is not sufficient to recommend using VS/VS Code, we also need to provide instructions on how to run Q# from other IDEs, even if they don't provide rich functionality, because there will be users who are used to their IDEs and don't want to switch.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 97e509ca-7867-e8f0-45f1-bb20d27ad232
* Version Independent ID: a6a281f8-53e0-8bdb-2b6e-8228fc2204f2
* Content: [Develop with Q# command line applications - Microsoft Quantum](https://docs.microsoft.com/en-us/quantum/install-guide/standalone)
* Content Source: [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/live/articles/install-guide/standalone.md)
* Product: **quantum**
* GitHub Login: @KittyYeungQ
* Microsoft Alias: **kitty**

### PR DESCRIPTION
**Pull request by gillenhaalb**
Tried to ease up on the language that made it sound like VS and VS Code were the only environments possible, and added instructions to each of the install pages. Also made some minor changes to make things more readable/symmetric between the pages.

Had to scrape from old branches for the specifics of installing the QDK not via the extensions, so definitely could use some verification that it's still correct (and still applies for the command line application, which didn't exist previously), @tcNickolas @cgranade.

Not sure how much, if any, detail should be added for working from any specific different editors. Figure it might be straightforward from these instructions on---especially since if someone's deliberately using a different editor, it's probably not their first rodeo---but any input there would be great too.

Fixes #883.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0134730a78e10200411d2c8199225e450b8f77ab/articles/install-guide/csinstall.md) | 93 | 86 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/182fdbbe-dd0c-42d1-be66-c03ca007f039_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0134730a78e10200411d2c8199225e450b8f77ab/articles/install-guide/pyinstall.md) | 90 | 81 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a2a4be7a-4ca1-4d3a-ad0b-cf1d98eee018_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0134730a78e10200411d2c8199225e450b8f77ab/articles/install-guide/standalone.md) | 85 | 65 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0a95f306-788f-4e89-9207-4b3331a0d043_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/dc1e9ecaac3979d6a03bfaa093c8748b510cb009/articles/install-guide/csinstall.md) | 93 | 86 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d0f5d0b9-47ad-4e5f-a003-dbf846d8ab97_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/dc1e9ecaac3979d6a03bfaa093c8748b510cb009/articles/install-guide/pyinstall.md) | 90 | 81 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/093b6bc5-23a3-4eab-bc81-e068e19b02a2_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/dc1e9ecaac3979d6a03bfaa093c8748b510cb009/articles/install-guide/standalone.md) | 85 | 65 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b09e7ad7-32db-4655-9c3f-4f6e5833fc01_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5c1181f0149491d00823e30a77dddcbcfe3e5710/articles/install-guide/csinstall.md) | 93 | 86 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f42f3042-a6fc-43a4-8915-d028bf61b801_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5c1181f0149491d00823e30a77dddcbcfe3e5710/articles/install-guide/pyinstall.md) | 90 | 81 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/32ca29e2-6c37-4f2b-8bdd-4f2153dcee1a_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5c1181f0149491d00823e30a77dddcbcfe3e5710/articles/install-guide/standalone.md) | 85 | 65 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/612a6cad-539d-481a-8dbb-b43ebb50bf55_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9e4096851f629bb6b37ecce7eb88796c65478822/articles/install-guide/csinstall.md) | 93 | 86 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/da9260dd-9943-4606-98d0-22aa1b8c763e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9e4096851f629bb6b37ecce7eb88796c65478822/articles/install-guide/pyinstall.md) | 90 | 81 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6c758364-0e03-49c5-8f08-c17f087e8933_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9e4096851f629bb6b37ecce7eb88796c65478822/articles/install-guide/standalone.md) | 85 | 65 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f6810ba4-a138-4f46-8ca0-4c818f186988_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/7aa1959c8658ad4ca6e9feedaa306548c7ba828e/articles/install-guide/csinstall.md) | 93 | 86 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0c65072a-9220-45b0-8c27-dee9ae715e7f_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/7aa1959c8658ad4ca6e9feedaa306548c7ba828e/articles/install-guide/pyinstall.md) | 90 | 81 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/87ee1841-bcec-4f6f-9a15-1846d3cf98c0_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/7aa1959c8658ad4ca6e9feedaa306548c7ba828e/articles/install-guide/standalone.md) | 87 | 65 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fb2c1f0c-2e96-47ab-85cb-588fa69124ae_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/install-guide/pyinstall.md [hunk 1]
```diff
@@ -41,11 +41,15 @@ Install the QDK to develop Python host programs to call Q# operations.
...
   
-1. While you can use Q# with Python in any IDE, we highly recommend using Visual Studio Code (VS Code) IDE for your Q# + Python applications. By using Visual Studio Code and the QDK Visual Studio Code extension you gain access to richer functionality.
+1. While you can use Q# with Python in any IDE, we highly recommend using Visual Studio Code (VS Code) IDE for your Q# + Python applications. Through the QDK Visual Studio Code extension you gain access to richer functionality such as warnings, syntax highlighting, project templates, and more.
+
+    If you would like to use VS Code:
 
     - Install [VS Code](https://code.visualstudio.com/download) (Windows, Linux and Mac)
     - Install the [QDK extension for VS Code](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode).
 
+    If you would like to use a different editor, the instrucitons above have you all set.
```
Comment (geduardo) [inline articles/install-guide/pyinstall.md:51]:
```suggestion
    If you would like to use a different editor, the instructions above have you all set. 
```

File: articles/install-guide/standalone.md [hunk 2]
```diff
@@ -78,6 +85,30 @@ To run the application:
 > [!NOTE]
 > If you have multiple projects within one Visual Studio solution, all projects contained in the solution need to be in the same folder as the solution, or in one of its sub-folders.  
 
+### [Command line or other editors](#tab/tabid-cmdline)
```
Comment (geduardo) [inline articles/install-guide/standalone.md:88]:
```suggestion
### [Other editors with the command line](#tab/tabid-cmdline)
```

File: articles/install-guide/standalone.md [hunk 3]
```diff
@@ -78,6 +85,30 @@ To run the application:
...
+    dotnet new console -lang Q# -o runSayHello
+    ```
+
+2. Navigate to the application directory:
+    ```dotnetcli
+    cd runSayHello
+    ```
+
+    This directory should now contain a file `Program.qs`, which is a Q# program that defines a simple operation to print a message to the console.
+
```
Comment (geduardo) [inline articles/install-guide/standalone.md:103]:
```suggestion
    This directory should now contain a file `Program.qs`, which is a Q# program that defines a simple operation to print a message to the console. You can modfiy this template with a text editor and overwrite it with your own quantum applications. 

```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/62526727e773d9c4dac90c0775f31504db9153bc/articles/install-guide/csinstall.md) | 93 | 86 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/93d5fb17-d639-4bd6-8454-4602398fb20b_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/62526727e773d9c4dac90c0775f31504db9153bc/articles/install-guide/pyinstall.md) | 93 | 81 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1ef7cab0-d783-4ef1-a16e-e1f5ad0b868f_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/62526727e773d9c4dac90c0775f31504db9153bc/articles/install-guide/standalone.md) | 85 | 66 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dd8c9fa8-09d9-4f75-ae50-9766895c539e_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/install-guide/standalone.md [hunk 4]
```diff
@@ -30,8 +30,15 @@ To configure Visual Studio:
 1. Download and install [Visual Studio](https://visualstudio.microsoft.com/downloads/) 16.3 or greater, with the .NET Core cross-platform development workload enabled.
 2. Download and install the [Microsoft QDK](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit).
 
+To install the QDK for another environment, enter in the command line:
 
-## Develop with Q# using VS Code
+    `dotnet new -i Microsoft.Quantum.ProjectTemplates`
```
Comment (gillenhaalb) [inline articles/install-guide/standalone.md:35]:
```suggestion
    dotnet new -i Microsoft.Quantum.ProjectTemplates
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a415cd51a26da8ea3e6bd3ed5b44df386a179c6d/articles/install-guide/csinstall.md) | 93 | 86 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c8dbb6f1-c010-4b5d-9354-b01d4a92007e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a415cd51a26da8ea3e6bd3ed5b44df386a179c6d/articles/install-guide/pyinstall.md) | 93 | 81 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3d394231-9479-4cf1-8cf7-9cbac26afd18_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a415cd51a26da8ea3e6bd3ed5b44df386a179c6d/articles/install-guide/standalone.md) | 85 | 66 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4fd3910f-fee6-4a6e-8185-f6269aa3169c_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/install-guide/standalone.md [hunk 5]
```diff
@@ -30,8 +30,15 @@ To configure Visual Studio:
 1. Download and install [Visual Studio](https://visualstudio.microsoft.com/downloads/) 16.3 or greater, with the .NET Core cross-platform development workload enabled.
 2. Download and install the [Microsoft QDK](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit).
 
+To install the QDK for another environment, enter in the command line:
 
-## Develop with Q# using VS Code
+    dotnet new -i Microsoft.Quantum.ProjectTemplates
```
Comment (gillenhaalb) [inline articles/install-guide/standalone.md:35]: @geduardo @cgranade @tcNickolas I'm getting some very odd behavior here. This line was giving me wild rendering problems when code blocked with triple ``'s, so I went to make it in-line with single quotes. Then, it renders as a full code block, but with the single quotes included. I remove the quotes entirely, and voila, it's rendering perfectly as a code block without any instructions telling it to do so. 

Is there some trick (or bug) in the markdown scheme that I'm missing here?

Comment (cgranade) [reply articles/install-guide/standalone.md:35]: I'm not sure why the ` ``` ` fences were failing, but indenting by four or more whitespace characters is an equivalent way of denoting literal code blocks, but comes with the disadvantage that you can't specify a language that way. The [DocFX Flavored Markdown reference](https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html?tabs=tabid-1%2Ctabid-a) has more detail.

(As a side note, you can now include cells from Jupyter Notebook files...?)

File: articles/install-guide/csinstall.md [hunk 6]
```diff
@@ -10,7 +10,9 @@ uid: microsoft.quantum.install.cs
 # Develop with Q# and .NET
 
 Q# is built to play well with .NET languages such as C# and F#.
-In this guide, we'll demonstrate how to use Q# with a host program written in a .NET language.
+In this guide, we demonstrate how to use Q# with a host program written in a .NET language.
+
+First we create the Q# library and .NET host, and then demonstrate how to call to Q# from the host.
```
Comment (cgranade) [inline articles/install-guide/csinstall.md:15]: Strictly speaking, the Q# project defines a quantum executable as entry points aren't allowed in libraries.
```suggestion
First we create the Q# application and .NET host, and then demonstrate how to call to Q# from the host.
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2d3c4fd8409cd36736b68935bac8c047e01423c8/articles/install-guide/csinstall.md) | 93 | 86 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/97542c10-180b-4a3a-8815-ffa73093e597_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2d3c4fd8409cd36736b68935bac8c047e01423c8/articles/install-guide/pyinstall.md) | 93 | 81 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2d9d9c9d-18d9-4325-9686-8c7b94a61047_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2d3c4fd8409cd36736b68935bac8c047e01423c8/articles/install-guide/standalone.md) | 85 | 66 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7f2aad19-484d-48af-b932-05b4787faae1_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e12fb4ddf2a5339773fae5e3d9e590dcef3c9aae/articles/install-guide/csinstall.md) | 93 | 86 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed0bdb96-2290-45a9-af9e-611e2810af9e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e12fb4ddf2a5339773fae5e3d9e590dcef3c9aae/articles/install-guide/pyinstall.md) | 93 | 81 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/22b574a4-f141-4098-bdba-01267f0bde88_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e12fb4ddf2a5339773fae5e3d9e590dcef3c9aae/articles/install-guide/standalone.md) | 85 | 66 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f1efbf71-d037-42dc-b678-7e7233157eb6_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e12fb4ddf2a5339773fae5e3d9e590dcef3c9aae/articles/install-guide/csinstall.md) | 93 | 86 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/388fdeac-75b1-4c02-912d-3a59526eb418_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e12fb4ddf2a5339773fae5e3d9e590dcef3c9aae/articles/install-guide/pyinstall.md) | 93 | 81 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e5bc365f-dace-4dd4-bb65-8ec5d7dadeed_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e12fb4ddf2a5339773fae5e3d9e590dcef3c9aae/articles/install-guide/standalone.md) | 85 | 66 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ebb08876-b1c9-478f-a89c-a73ef41150c2_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/quickstarts/install-command-line.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8eba1a0850107975e4cd2170038bd2c31f7356cb/articles/quickstarts/install-command-line.md) | 85 | 66 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c192679c-20b0-476d-8e54-70c006ad50cf_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/install-cs.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8eba1a0850107975e4cd2170038bd2c31f7356cb/articles/quickstarts/install-cs.md) | 93 | 86 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7c18c3bb-7ea6-46f5-b0e2-4b20ee14cbce_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/install-python.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8eba1a0850107975e4cd2170038bd2c31f7356cb/articles/quickstarts/install-python.md) | 93 | 81 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f5ed6979-b806-45fe-8ed3-4efabd398a8a_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/quickstarts/install-command-line.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a6ed443d4b6d381b3974cfba2bd03b1edc3ec648/articles/quickstarts/install-command-line.md) | 85 | 66 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/86e0e457-8b0f-45e6-91fb-f770125c8a9e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/install-cs.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a6ed443d4b6d381b3974cfba2bd03b1edc3ec648/articles/quickstarts/install-cs.md) | 93 | 86 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/81e4ec72-4060-44f6-8f9c-882cfc912455_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/install-python.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a6ed443d4b6d381b3974cfba2bd03b1edc3ec648/articles/quickstarts/install-python.md) | 93 | 81 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/85705f63-4e7b-4799-bfe8-17caef361b9d_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**